discovery-provider/src/utils/cache_solana_program.py

```diff
@@ -27,18 +24,16 @@ def cache_latest_sol_db_tx(
     redis: Redis, cache_key: str, latest_tx: CachedProgramTxInfo
 ):
     try:
-        redis_set_json_and_dump(redis, cache_key, latest_tx)
+        redis.set(cache_key, json.dumps(latest_tx))
     except Exception as e:
         logger.error(
             f"cache_solana_program.py | Failed to cache key {cache_key} latest processed transaction {latest_tx}, {e}"
         )
         raise e
 
 
-def get_latest_sol_db_tx(redis: Redis, cahce_key: str):
-    latest_sol_db: Optional[CachedProgramTxInfo] = redis_get_json_cached_key_or_restore(
-        redis, cahce_key
-    )
+def get_latest_sol_db_tx(redis: Redis, cache_key: str):
+    latest_sol_db: Optional[CachedProgramTxInfo] = json.loads(redis.get(cache_key))
```

Comment:
I think previously this would return None if there is no redis key, but maybe that will never happen? Does local dev for this work? Maybe fine
Calling json.loads on none causes an err.

Reply:
Yeah, will make this change. I think local is broken for another reason, will ping abt that.